### ISSUE TITLE
Crossword:  scrolling makes highlighted words vanish

### ISSUE THREAD

**Issue by Golpette** [AndrewRigg/Crossword#94]
Sorry Andy, I didn't see this before doing all the merges... whenever I scroll on crossword, the entered words now vanish.

I'm 99% sure this couldn't have been me and it didn't used to do this - do you know what you might have changed?

**Comment by Golpette** [comment]
Maybe this will help you track it:

weirdly if i zoom, then the entered words are "locked in" and never vanish

but if I enter a word then immediately scroll it does

**Comment by AndrewRigg** [comment]
I think I changed the grid location of stuff down at the bottom.  Could it be to do with that?  In draw grid, all the bounds were changed from  
hintArea.setBounds(squareSize, squareSize, squareSize * (x - 2), squareSize * (y - 2));
to
hintArea.setBounds(initialSquareSize/2, initialSquareSize/2, squareSize * (x - 2), squareSize * (y - 2));
for each entity.  This was an attempt to stop the grid jumping up or down by a small amount every time it was resized.  (As the former changed with size but the latter was a final constant)
